src/PostgREST/Auth/JwtCache.hs

```diff
@@ -25,19 +27,29 @@ import PostgREST.Error      (Error (..))
 
 import Protolude
 
-newtype JwtCacheState = JwtCacheState
-  { jwtCache :: C.Cache ByteString AuthResult
+-- | JWT Cache and IO action that triggers purging old entries from the cache
+data JwtCacheState = JwtCacheState
+  { jwtCache   :: C.Cache ByteString AuthResult
+  , purgeCache :: IO ()
   }
 
 -- | Initialize JwtCacheState
 init :: IO JwtCacheState
 init = do
-  cache <- C.newCache Nothing -- no default expiration
-  return $ JwtCacheState cache
+  cache <- C.newCache Nothing
```

Comment:
No need to remove this comment.
```suggestion
  cache <- C.newCache Nothing -- no default expiration
```

Reply:
`newCache` function from our cache library comes with a parameter for default expiration. `Nothing` here means that we don't have default expiration for our cache entries. We add expiration at time of insertion.

Reply:
`newCache` function from our cache library comes with a parameter for default expiration. `Nothing` here means that we don't have default expiration for our cache entries. We add expiration at time of insertion.